src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure implements AreaCalculetion {
+    private double sideB;
+    private double sideA;
+
+    public Rectangle(double sideA, double sideB, String color, String name) {
+        this.sideA = sideA;
+        this.sideB = sideB;
+    }
+
+    public double getSideA() {
+        return sideA;
+    }
+
+    public double getSideB() {
+        return sideB;
+    }
+
+    @Override
+    public double getArea() {
+        return getSideA() * getSideB();
+    }
+
+    @Override
+    public String toString() {

```

Comment:
Use only English in messages/code. The `toString` method currently returns a string with the word 'Figure', which is English, but ensure that all user-facing messages are in English to maintain consistency.